rest_api/actix_web_1/Cargo.toml

```diff
@@ -25,10 +25,10 @@ description = """\
 [dependencies]
 actix-web = { version = "1", optional = true }
 futures = { version = "0.1", optional = true }
-log = { version = "0.4", optional = true }
```

Comment:
These features should stay optional, and instead get pulled in if "service" or "admin-service" is pulled in.